src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1071,56 +1071,6 @@ public async Task GetAsync_ManyDifferentResponseHeaders_ParsedCorrectly(string n
                 dribble ? new LoopbackServer.Options { StreamWrapper = s => new DribbleStream(s) } : null);
         }
 
-        [Theory]
```

Comment:
Previously the GetAsync_TrailingHeaders_Ignored test would run for WinHttpHandler and netfx, validating that they didn't blow up if trailers were provided.  With this change, that validation is no longer being performed.

Reply:
I moved it to PlatformHandlerTest and made Windows specific to reduce noise.